src/cli/domain/local.js

```diff
@@ -32,36 +33,39 @@ module.exports = function(){
     },
     getComponentsByDir: getComponentsByDir(),
     getLocalNpmModules: getLocalNpmModules(),
-    init: function(componentName, templateType, callback){
+    init: function(componentName, templateType, options, callback){
 
       if(!validator.validateComponentName(componentName)){
         return callback('name not valid');
       }
 
-      if(!validator.validateTemplateType(templateType)){
-        return callback('template type not valid');
-      }
-
-      try {
-
-        const pathDir = '../../components/base-component-' + templateType,
-          baseComponentDir = path.resolve(__dirname, pathDir),
-          npmIgnorePath = path.resolve(__dirname, pathDir + '/.npmignore');
+      // LEGACY TEMPLATES
```

Comment:
Can we move this to own file like `init-legacy-templates.js` so that we can make this comment redundant?